types/meteor-files/tslint.json

```diff
@@ -0,0 +1,6 @@
+{ 
+  "extends": "dtslint/dt.json",
+  "rules": {
```

Comment:
This rule needs to be removed and the error needs to be fixed

Reply:
The package is not in npm. It is a Meteor js package, hosted here https://atmospherejs.com/ostrio/files

Reply:
When you remove it , lint will show you how to fix the error